src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -207,6 +207,7 @@ enum AppStrings {
 		static let refreshedFormat = NSLocalizedString("ExposureDetection_Refreshed_Format", comment: "")
 		static let refreshedNever = NSLocalizedString("ExposureDetection_Refreshed_Never", comment: "")
 		static let refreshingIn = NSLocalizedString("ExposureDetection_RefreshingIn", comment: "")
```

Comment:
What's the difference between "RefreshingIn" and "RefreshIn" ? seems a little bit confusing to me there.

Reply:
slightly different text strings. agree, its not too clear tho